tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -2163,6 +2180,12 @@ Status FragmentMetadata::write_generic_tile_offsets(Buffer* buff) const {
   return Status::Ok();
 }
 
+Status FragmentMetadata::write_array_schema_name(Buffer* buff) const {
+  uint64_t size = array_schema_name_.size();
```

Comment:
error out if `array_schema_name_` is empty

Reply:
added